### ISSUE TITLE
llama.cpp / GGUF support

### ISSUE THREAD

**Issue by sammcj** [allenai/OLMoE#7]
It would be great to see OLMoE/OlmoeForCausalLM Llama.cpp/GGUF support.

Really neat project!

**Comment by AmitKKhanchandani** [comment]
+1, need to try this with ollama :)

**Comment by Bobetele** [comment]
yes

**Comment by Muennighoff** [comment]
won't have bandwidth to do this, but if anyone is interested, that'd be amazing!

**Comment by MrDowntempo** [comment]
Yeah, this is hard to work with if it isn't in GGUF format to run locally, or available from Ollama directly. I'm looking into how to serve from Safetensors but not a lot of servers support that.

**Comment by Muennighoff** [comment]
also cc @2015aroras

**Comment by 2015aroras** [comment]
See https://github.com/ggerganov/llama.cpp/pull/9462

**Comment by Meshwa428** [comment]
Any updates?

**Comment by Meshwa428** [comment]
> See https://github.com/ggerganov/llama.cpp/pull/9462

It still isn't merged 😞

**Comment by baileykuehl** [comment]
Hi, thanks again for the inquiry! We’re currently working on closing out old tickets, so we’re closing this out for now, but if you require a follow-up response, please re-open and we will get back to you!
